helpers/resourceHints.js

```diff
@@ -16,28 +15,28 @@ const factory = globals => {
             return `<link rel="dns-prefetch preconnect" href="${host}" crossorigin>`;
         }
 
-        var hosts = [];
-
         // Add cdn
         const siteSettings = globals.getSiteSettings();
-        const cdnUrl = siteSettings['cdn_url'] || '';
-        if (cdnUrl != '') {
-            hosts.push(cdnUrl);
-        }
+        const hosts = siteSettings['cdn_url'] ?  [siteSettings['cdn_url']] : [];
 
         // Add font providers
-        const fontProviders = _.keys(getFonts('providerLists', globals.getThemeSettings(), globals.handlebars));
-        _.each(fontProviders, function(provider) {
-            if (typeof fontResources[provider] !== 'undefined') {
-                hosts = hosts.concat(fontResources[provider]);
+        const providers = Object.keys(getFonts('providerLists', globals.getThemeSettings(), globals.handlebars));
+
+        for (let i = 0; i < providers.length; i++) {
+            if (typeof fontResources[providers[i]] !== 'undefined') {
+                hosts.push(...fontResources[providers[i]]);
             }
-        });
+        }
+
+        for (let j = 0; j < hosts.length; j++) {
```

Comment:
Perhaps instead of a second loop, we should just do this as part of the first loop?

Reply:
We don't have individual access to each host in the first.